scripts/resolve_network.py

```diff
@@ -23,28 +23,32 @@
 
 def add_H2(n, snakemake):
 
-    area = snakemake.config['area']
     year = snakemake.wildcards.year
-    policy = snakemake.wildcards.policy
     country_targets = snakemake.config[f"h2_target_{year}"]
     
     for country, target in country_targets.items():
-        
-        node = geoscope(n, country, area)['node']
-    
-        # remove electricity demand of electrolysis
-        load_i = pd.Index([f"{node} electrolysis demand"])
```

Comment:
Just as an explanation:
this was an option we had for the two step optimisation. We did this as a sensitivity, in the first step we didn't model the hydrogen production but added to the electricity demand in the background system the elecetricity demand of the electrolysis to produce the set hydrogen demand. In the second step we removed this exogenous set electricity demand of the electrolysis and added the actual hydrogen production. In our cases `load_i` is always going to be empty